src/main/java/no/nav/fo/veilarbdialog/kvp/KontorsperreFilter.java

```diff
@@ -1,22 +1,18 @@
 package no.nav.fo.veilarbdialog.kvp;
 
-import lombok.SneakyThrows;
-import no.nav.apiapp.security.PepClient;
+import lombok.RequiredArgsConstructor;
+import no.nav.fo.veilarbdialog.auth.AuthService;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.stereotype.Component;
 
-import javax.inject.Inject;
-
-import static no.nav.apiapp.util.StringUtils.nullOrEmpty;
-
 @Component
+@RequiredArgsConstructor
 public class KontorsperreFilter {
 
-    @Inject
-    PepClient pepClient;
+    private final AuthService auth;
 
-    @SneakyThrows
-    public boolean harTilgang(String enhet) {
-        return nullOrEmpty(enhet) || pepClient.harTilgangTilEnhet(enhet);
+    public boolean harTilgang(String ident, String enhet) {
+        return StringUtils.isEmpty(enhet) || auth.harVeilederTilgangTilEnhet(ident, enhet);
```

Comment:
Fungerer det her for eksternbrukere nå da?

Reply:
Vel, i `Pep` fra fellesbiblioteket så har du jo kun anledning til å spørre harVeilederTilgangTilEnhet(...).

Hva foreslår du å bruke fra `Pep` i stedet?

Reply:
Hmm, jeg tror du må først sjekke tilgang til person(`harTilgangTilPerson`) og tilgang til kvp enhet tror jeg ( `harTilgangTilEnhetMedSperre`). Men ville spurt Kari-Anne for eksempel om hvordan det skal fungere fordi det er jeg ikke sikkert på. Eller kanskje @hakonph vet?

Reply:
Håkon sier at det er riktig så da resolver jeg denne